### PR TITLE
Fixes https://github.com/NativeScript/nativescript-cli/issues/1123

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: lib/services/usb-livesync-service.ts [hunk 1]
```diff
@@ -67,8 +67,11 @@ export class UsbLiveSyncService extends usbLivesyncServiceBaseLib.UsbLiveSyncSer
 				}).future<boolean>()();
 			};
 
-			let notRunningiOSSimulatorAction = (): IFuture<void> => {
-				return this.$platformService.deployOnEmulator(this.$devicePlatformsConstants.iOS.toLowerCase());
+			let notRunningiOSSimulatorAction = (): IFuture<boolean> => {
+				return (() => {
+					 this.$platformService.deployOnEmulator(this.$devicePlatformsConstants.iOS.toLowerCase()).wait();
+					 return false;
```
Comment (rosen-vladimirov) [inline lib/services/usb-livesync-service.ts]: what's the point of returning false every time?

Comment (Fatme) [inline lib/services/usb-livesync-service.ts]: https://github.com/telerik/mobile-cli-lib/pull/506/files#diff-c8066f499a9cad66c8bae208dd04b18cR145. The problem is that `livesync` command doesn't need to sync the application if the simulator is not running, but the `test` command needs to perform livesync after the simulator get started.

Comment (teobugslayer) [inline lib/services/usb-livesync-service.ts]: I suggest to add a comment in the code explaining this.

**Comment by rosen-vladimirov** [comment]
:+1:  after green build

**Pull request merged**